docs/what_is_data_science/Benefits of Data Science in the NHS.md

```diff
@@ -0,0 +1,9 @@
+# Benefits of Data Science in the NHS
+
+Data Science can be helpful in [solving a number of problems](./index.md). Specifically, this could lead to benefits like:
+
+- **better resource planning** (e.g. [improving bed allocation using AI](../our_work/casestudy-bed-allocation.md))
+- **increased responsiveness to demand and seasonal pressures** (e.g. A&E demand prediction)
+- **improved data quality** (leading to higher quality analytics and decision making) through better data linkage.
```

Comment:
i dont think (leading to higher quality analytics and decision making) needs to be in brackets, it works as part of the main sentence and we dont want to minimise its importance!

Reply:
Rejigged it a bit.